cost/aws/instance_cloudwatch_utilization/README.md

```diff
@@ -1,83 +1,85 @@
-- The **Exclusion Tag Key** parameter is a string value.  Supply the Tag Key only.  Tag Values are not analyzed and therefore are not need.  If the exclusion tag key is used on an Instance, that Instance is presumed to be exempt from this policy.
-- This policy sets the tag defined in the **Action Tag Key:Value** parameter on the underutilized instances that were identified.
-- If you get an **N/A** in a field you will need to install the [CloudWatch Agent](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/Install-CloudWatch-Agent.html) on the instance to get those metrics. 
-- This policy only pulls running instances, as it is unable to get correct monitoring metrics from instances in other states.
+The **Exclusion Tag Key** parameter is a string value.  Supply the Tag Key only.  Tag Values are not analyzed and therefore are not need.  If the exclusion tag key is used on an Instance, that Instance is presumed to be exempt from this policy.
 
-#### Windows Support
+Inefficient Instances are resized to the next smaller size within the same class.  If the instance is already on the smallest size for the class it will not be resized. Resize only occurs after approval
 
-To enable windows support you will need to add the following to your cloudwatch config.json and restart cloudwatch agent
-```json
-	"metrics": {
-		"append_dimensions": {
-			"AutoScalingGroupName": "${aws:AutoScalingGroupName}",
-			"ImageId": "${aws:ImageId}",
-			"InstanceId": "${aws:InstanceId}",
-			"InstanceType": "${aws:InstanceType}"
-    }
-  }
-```
-#### Input Parameters
+If you get an **N/A** in a field you will need to install the [CloudWatch Agent](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/Install-CloudWatch-Agent.html) on the instance to get those metrics.
+
+## Policy Actions
+
+- Sends an email notification
+- Downsizes instances after approval
+
+## Prerequisites
+
+This policy requires the AWS IAM or AWS STS Credential. When applying the policy select the appropriate credentials
+from the list for your tenant. If such credential doesn't exist please contact your cloud admin to create the Credential.
+
+The credential must contain the value *AWS* in the Provider field.
+Refer to our documentation for more details on the [Credential Service](https://docs.rightscale.com/credentials/)
+
+## Input Parameters
 
 This policy has the following input parameters required when launching the policy.
 
-- *Email addresses of the recipients you wish to notify* - A list of email addresses to notify
-- *Average used memory percentage* - Utilization below this percentage will raise an incident to tag the instance. Providing -1 will turn off this metric for consideration.
+- *Email addresses of the recipients you wish to notify*
+- A list of email addresses to notify- *Average used memory percentage*
+- Utilization below this percentage will raise an incident to tag the instance. Providing -1 will turn off this metric for consideration.
```

Comment:
these bullets are not accurate. it looks like the lines merged.

Reply:
Updated README file by considering above comment